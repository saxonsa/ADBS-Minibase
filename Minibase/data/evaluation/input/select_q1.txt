Q(x,y,z) :- R(x, y, z), x = 8