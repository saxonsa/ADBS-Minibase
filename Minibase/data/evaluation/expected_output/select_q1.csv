8, 1, 'mlpr'
8, 9, 'rl'
8, 9, 'ppls'
